database/migrations/2024_06_19_072618_create_countries_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('countries', function (Blueprint $table) {
```

Comment:
@dhrumin-7span Please add the index on the below fields
- name
- phonecode
- numcode

Reply:
Added indexing on this fields.